scripts/env/env.rb

```diff
@@ -4,45 +4,44 @@ module Environment
   def self.rust
     config = Config.new
     if config.get_rust_version == 'stable'
-      Reporter.add(Jobs::Skipped, Owner::Env, 'Skip checking of rust version for stable', '')
+      Reporter.skipped('Env', 'Skip checking of rust version for stable', '')
       return
     end
     output = `rustc -V`
     if output.include? config.get_rust_version
-      Reporter.add(Jobs::Skipped, Owner::Env,
-                   "Target version of rust (#{config.get_rust_version}) already installed", '')
+      Reporter.skipped('Env', "Target version of rust (#{config.get_rust_version}) already installed", '')
       return
     end
     Shell.sh "rustup install #{config.get_rust_version}"
     Shell.sh "rustup default #{config.get_rust_version}"
```

Comment:
@marcmo, module `Environment` is used not only to check envvars, but also to setup correct envvars. By default, we are using `stable` version of rust... But very rarely we have cases when we have to stick with some specific version of rust. It's related to our dependencies like `node-binding` and `wasm`.

Reply:
In any way we have a duplicate of this functionality here `scripts/tasks.rb`
```
namespace :env do
  desc 'Install target version of rust'
  task :rust do
    config = Config.new
    sh "rustup install #{config.get_rust_version}"
    sh "rustup default #{config.get_rust_version}"
  end
end
```
I think this module `Environment`  is a still suitable place to check version and install it if it's needed. At least because module has flag `@@checked` and can be called multiple times without recheching and reinstalling. 

But of cause duplicate (from  `scripts/tasks.rb`) should be referred to `Environment`  or removed at all